src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
```

Comment:
`result` is a bad name, the variable should better describe what is it. For example, you name it `statesHistory` or something similar